convert/src/vertex_to_bd/badgerdoc_to_vertex_use_case.py

```diff
@@ -0,0 +1,84 @@
+import tempfile
+from pathlib import Path
+from typing import NamedTuple
+
+from src.vertex_to_bd.labelstudio_format import LabelStudioFormat
+
+from .models import S3Path
+from .models.bd_annotation_model import BadgerdocAnnotation
+from .models.bd_tokens_model import Page
+
+
+class BadgerdocData(NamedTuple):
+    page: Page
+    annotation: BadgerdocAnnotation
+
+
+class BDToVertexConvertUseCase:
+    def __init__(
+        self,
+        s3_client=None,
```

Comment:
We should have default value here, especially for s3_client. It has to be mandatory

Reply:
Actually, we don't need it here at all. Methods of this class should accept s3_client on their own. And location and name of this class a bit confusing with the actual stuff it does

Reply:
>We should have default value here, especially for s3_client. It has to be mandatory
Sure, it was for development purposes, thanks for catching.

>Actually, we don't need it here at all. Methods of this class should accept s3_client on their own. 
I think passing storage in an init method is ok. We reduce the amount of variables in method signatures. I see only one disadvantage: we can't upload and download from different sources. 

>And location of this class a bit confusing with the actual stuff it does
Probably, I think we should create another issue

>And name of this class a bit confusing with the actual stuff it does
In my opinion, the name is descriptive. It does everything to convert from BD to Vertex format.